src/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -136,9 +137,7 @@ export class OverviewRoute extends React.Component<OverviewRouteProps, State> {
 
         {!hideMainOverviewSection && <Spacer mb={4} />}
 
-        <Box>
-          <ArtistCollectionsRail artistID={artist._id} />
-        </Box>
+        <Box>{isClient && <ArtistCollectionsRail artistID={artist._id} />}</Box>
```

Comment:
Without always including the containing `<Box>`, the rehydration when this content appears seemed to screw up the rest of the page.